blueprint/core/blueprint_View.cpp

```diff
@@ -265,4 +265,24 @@ namespace blueprint
             parent->dispatchViewEvent(getViewId(), eventType, e);
     }
 
+    void View::exportMethod(const juce::String &method, juce::var::NativeFunction fn)
+    {
+        JUCE_ASSERT_MESSAGE_THREAD
+        nativeMethods[method] = std::move(fn);
+    }
+
+    juce::var View::invokeMethod(const juce::String &method, const juce::var::NativeFunctionArgs &args)
+    {
+        JUCE_ASSERT_MESSAGE_THREAD
+        auto it = nativeMethods.find(method);
+
+        if (it != nativeMethods.end())
+        {
+            return it->second(args);
+        }
+        else
```

Comment:
Nit: we don't need the `else` clause here, because we `return` from the `if`. So can just

```
if (it != nativeMethods.end())
    return it->second(args);

throw std::logic_error....
```

Reply:
Yup no worries. think I'd been watching a talk advocating explicit `else` and got overly inspired!